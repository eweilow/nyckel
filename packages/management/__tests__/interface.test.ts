describe("public API", () => {
  const obj = require("../");

  it("should be correct", () => {
    expect(obj).toMatchSnapshot();
  });
});
