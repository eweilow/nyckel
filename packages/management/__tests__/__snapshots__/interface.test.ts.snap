// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`public API should be correct 1`] = `
Object {
  "getManagementToken": [Function],
  "getManagementUserInfo": [Function],
  "getManagementUserPermissions": [Function],
  "getManagementUserRoles": [Function],
}
`;
